src/main/java/com/easypost/model/CarrierAccount.java

```diff
@@ -58,11 +58,19 @@ public CarrierAccount update(Map<String, Object> params, String apiKey) throws E
 
 		// delete
 		public void delete() throws EasyPostException {
-			this.delete(null);
+			this.delete(null, null);
 		}
-		public void delete(String apiKey) throws EasyPostException {
+		public void delete(String apiKey, String placeholder) throws EasyPostException {
```

Comment:
This is tricky bc of the two methods for delete() and delete(id (and there subsequent delete functions which are called). I could just rename the added function to delete_id(id) to get rid of this String placeholder, but I thought that it would be more convenient to be able to call delete() and delete(id).